rfcs/formats/identity.md

```diff
@@ -0,0 +1,203 @@
+
+**authentication** - The authentication verification relationship must point to the group controlled verification method of the Network DID.
+
+**networkParticipants** - This is not a core property of the DID specification. This is a special property required for Network DIDs that specify the members in the network. It is a list of DIDs of the individual members.
+
+### Multi-signature based verificationMethod
+
+A network DID must be controlled jointly by the network's participant units for operations such as creation of the DID, and its updation with changing network structure, and for other operations such as rotating keys, updating verification methods etc.. To enable this group control we introduce a verificationMethod that uses multi-signatures and policies.
+
+The verificationMethod will have the following properties:
+
+**id** - This will be the Network DID followed by a fragment forming a DID URI that points to this verification method. Eg: `did:<iin_name>:<network_name>#unique_verificationMethod_name`
+
+**type** - `BlockchainNetworkMultiSig`
+
+**controller** - Controller of this DID must be the network itself, and thus its value should be same as the Network DID.
+
+**Verification Materials for BlockchainNetworkMultiSig:**
+
+**multisigKeys** - Map of participant DID to its verificationMethod. This contains one entry for each participant unit in `networkParticipants` list. The key is the participant unit's DID and the value is a DID URI to the verification method of the participant DID.
+
+
+Eg. If `network_participant_1` has a DID `did:<iin_name>:<network_participant_1>`, then `multisigKeys` will contain one verification method of the DID Document like `"did:<iin_name>:<network_participant_1>":"did:<iin_name>:<network_participant_1>#verificationMethod1"`
+
+
+For creation of Network DID, the IIN registry must authenticate the DID creation request based on the `BlockchainNetworkMultiSig` verification method. This implies that the request must be authenticated based on each of the verificationMethods specified in  `multisigKeys`.
+
+
+> NOTE on extensibility: The `networkParticipants`  lets any DID to be specified, with a verification mehtod in `BlockchainNetworkMultiSig.multisigKeys`.  Thus another Network DID can also be represented as a participant unit of a network. This can be very useful in certain use cases where different blockchain networks may be stakeholders of a bigger network.
+
+
+
+**updatePolicy** - Defines a combination of participant units that must sign and authenticate a Network DID updation request. This `updatePolicy` has the condition format as specified in [`verifiablecondition2021`](https://w3c.github.io/did-spec-registries/#verifiablecondition2021) verification method. It can be used to combine different verification methods together to form conjugated conditions such as logical operations, thresholds, weighted thresholds, relationships and a delegation to external verification methods.
```

Comment:
There are two security guarantees this scheme would ideally offer, provided of course that the network's DID address is first obtained in a trusted manner: 1) that the network as a whole endorses the membership of every participant listed in the network DID document 2) conversely, that each member endorses its inclusion as a member in said network. Are we sure that the current scheme guarantees the latter? Two things to keep in mind to support this are: a) with the creation of a network's DID document there would ideally be a set of signatures from the resolved identities of the participants, each endorsing the network DID document b) during updates, if a new member is joining, it should sign the updated DID document with its DID in the list of members c) similarly, when a member leaves a network it would need to endorse an updated DID that omits it from the list of network participants. The latter two would require that `updatePolicy` to be dynamic and fine-grained (specific policies for targeted actions such as for additions/removals)

Reply:
The updatePolicy is discussed in a separate file now:
https://github.com/ghoshbishakh/weaver-dlt-interoperability/blob/rfc-identity/rfcs/models/identity/identity-update-policy.md